docs/tutorial/02-hello-world.mdx

```diff
@@ -26,42 +26,59 @@ A smart contract is a program that verifies and executes the performance of a co
 Programs that run on blockchains are commonly referred to as smart contracts because they mediate important functionality (such as currency)
 without having to rely on a central authority (like a bank).
 
-Before you get started with Flow, you need to understand how accounts and transactions are modeled.
+Before you get started with Flow, you need to understand how accounts and transactions are modeled on Flow and in Cadence.
 
 ## Accounts and Transactions
 
 ---
 
 Like most other blockchains, the programming model in Flow is centered around accounts and transactions.
-All persistent state is stored in [accounts](/cadence/language/accounts).
+All state that persists permanentely is stored in [accounts](/cadence/language/accounts)
+and there is only one type of account (all accounts can be for any number of users, smart contracts, and data)
+
 The interfaces to this state (the ways to interact with it) are also stored in accounts.
-All code execution takes place within transactions, which are blocks of code that are submitted by external users to interact with the persistent state,
+All code execution takes place within [transactions](/cadence/language/transactions), which are blocks of code
+that are authorized and submitted by external users to interact with the persistent state,
 which includes directly modifying account storage.
 
-Each user has an account controlled by one or more private keys.
-This means that support for accounts/wallets with [multiple controllers](https://www.coindesk.com/what-is-a-multisignature-crypto-wallet) is built into the protocol by default.
+### What is an account?
+
+Each user has an account controlled by one or more private keys with configurable weight.
+This means that support for accounts/wallets with [multiple controllers](https://www.coindesk.com/what-is-a-multisignature-crypto-wallet)
+is built into the protocol by default.
 
 An account is divided into two main areas:
 
 1. The first area is the [contract area](/cadence/language/accounts).
    This is the area that stores any number smart contracts that contain type definitions, fields,
    and functions that relate to a common functionality.
    This area also holds contract interfaces, which are basically program guidelines that other contracts can import and implement.
-   This area cannot be directly read from in a transaction, unless the transaction is just reading the code deployed to an account.
+   This area cannot be directly accessed in a transaction unless the transaction is just reading a copy of the code deployed to an account.
    The owner of an account can directly add, remove, or update/overwrite contracts that are stored in it.
 
 2. The second area is the account filesystem. This area is where an account stores the objects
```

Comment:
I didn't realize that we were referring to account storage as "account filesystem" here.
I think we should use the term "account storage" everywhere, though I like and we should keep the filesystem analogy below.

```suggestion
2. The second area is the account storage. This area is where an account stores the objects
```

Reply:
The reason I was using filesystem is because I didn't want people to get that mixed up with the `/storage/` domain, but I guess you're right that is probably doesn't make sense to call it a filesystem.

Reply:
Oh, I see now. Yeah, it's hard to find a good term for the `/storage` part of account storage that actually stores the objects. Maybe something like "object storage", as it stores objects, not files? In the same vein, the other areas, `/private` and `/public`, could be maybe named "link storage" (or "capability storage"?)?